lib/core/client.js

```diff
@@ -58,6 +70,25 @@ class NightwatchAPI {
     return browserNames.includes(browser);
   }
 
+  __isPlatformName(platform) {
+    const lowerCasePlatformName = this.platformName.toLowerCase();
+    const platformNames = [this.platformName, lowerCasePlatformName];
+
+    return platformNames.includes(platform);
```

Comment:
If character case is unimportant in this scenario, I think this function might be better written as below.
```
__isPlatformName(platform) {
    return this.platformName.toLowerCase() === platform.toLowerCase() 
}
```